CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ Fixed: Seeds with more than 3 starting Energy Parts can be exported and played.
 
 - Added: Use Pseudo Wave Beam (Intermediate) with Diffusion Beam to break the Early Gravity Blob through the wall.
 - Removed: Using Water Bomb Jumps to reach the Blob Alcove in Gravity Suit Tower.
+- Fixed: The Early Gravity Speed Booster puzzle now correctly requires Speed Booster Conservation set to Beginner.
```

Comment:
Aren't we supposed to list the changes with the prefix tag sorted alphabetically?